build_tools/ci/cpu_comparison/run.py

```diff
@@ -1849,6 +1867,45 @@ def __init__(self):
                 "transpose_a": False,
                 "transpose_b": False,
                 "skip_numerics": True,
+                "tile_pipeline": "pack-peel",
```

Comment:
Why not change all the tests in the "performance" section to use `pack-peel-4-level-tiling`?

Reply:
They are not all functional and to compare `pack-peel` and `pack-peel-4-level-tiling` for the functional shapes. Once all shapes are functional and more performant, I think we can deprecate the old `pack-peel`.